build.gradle

```diff
@@ -69,3 +69,20 @@ ext {
 dependencies {
     implementation 'junit:junit:4.12'
 }
+
+// updateVersion: Task to auto increment to the next development iteration
+task updateVersion {
+    onlyIf { System.getProperty('newVersion') }
```

Comment:
Where this `newVersion` will be set ? Where and when will this update version task will be executed?

Reply:
Example PR: https://github.com/prudhvigodithi/ml-commons/pull/2/files